homeassistant/components/meteoswiss/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "domain": "meteoswiss",
+  "name": "MeteoSwiss",
+  "codeowners": ["@albertomontesg"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/meteoswiss",
+  "homekit": {},
+  "iot_class": "cloud_polling",
+  "requirements": [
+    "meteoswiss-async @ git+https://github.com/albertomontesg/meteoswiss-async.git@main"
```

Comment:
Before being able to merge, this needs to be changed to the package name on PyPI.